recipes/gmmx/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "gmmx" %}
+{% set version = "0.5" %}
+{% set python_min = "3.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/gmmx-{{ version }}.tar.gz
+  sha256: ce0a7ee0ac633e0e528d60d1dc96a9d99a0d5b27bacb2e023e22ccfc211b0669
```

```suggestion
  sha256: d2141f82e98fc785b430d9888a7a237f6b7c60a09cc4d53d80c7028ba3ae017d
```

(that's coming from `openssl sha256 <target>`)